ckanext/security/mailer.py

```diff
@@ -30,16 +33,32 @@ def send_reset_link(user):
     mail_user(user, subject, body)
 
 
-def notify_lockout(user):
+def _build_footer_content(extra_vars):
+    custom_path = config.get('ckanext.security.brute_force_footer_path')
+    if (custom_path and os.path.exists(custom_path)):
+        log.warning('Overriding brute force lockout email footer with {}'.format(custom_path))
+        with open(custom_path, 'r') as footer_file:
+            footer_content = footer_file.read()
+        env = config['pylons.app_globals'].jinja_env
+        template = env.from_string(footer_content)
+        return '\n\n' + template.render(**extra_vars)
```

Comment:
Since the 'template' we are using to override isn't registered with this module initially, jinja2 doesn't have it loaded as a known template. This method reads the file and creates a template on the fly, then renders it. Part of this is heavily stolen from the base controller `render_jinja2` method.

Reply:
So the new template you created here `security/emails/lockout_footer.txt` is cached because it's part of this module? Whereas the custom one will not be (at least guaranteed) cached?

Reply:
Yes, that's true. Given the hopefully infrequent and asynchronous nature of lockout emails, I doubt that caching should be too big an issue in this case.